docs/product-assembly_pam-on-openshift/src/main/asciidoc/environment-authoring-single-proc.adoc

```diff
@@ -0,0 +1,35 @@
+[id='environment-authoring-single-proc']
+= Deploying a single authoring environment
+
+To deploy a single authoring environment, use the `rhpam-70-authoring.yaml` template file. You can download this file from `{URL_BASE_GITHUB_PAM}/templates`.
+
+.Procedure
+
+. Use one of the following methods to deploy the template:
+.. In the OpenShift Web UI, select *Add to Project >  Import YAML / JSON* and then select or paste the `rhpam-70-authoring.yaml` file. In the *Add Template* window, ensure *Process the template* is selected and click *Continue*.
+.. To use the OpenShift command line console, prepare the following command line:
++
+[subs="verbatim,macros"]
+----
+oc process -n <project-name> -f <template-path>/hpam-70-authorimg.yaml -v ATTRIBUTE=value -V ATTRIBUTE=value... | oc create -n <project-name> -f -
```

Comment:
hpam-70-authorimg.yaml -> rhpam-70-authoring.yaml
-v ATTRIBUTE=value -V ATTRIBUTE=value -> -p PARAMETER=value -p PARAMETER=value

Additionally, I know some prefer the "oc process" syntax in order to create and save a file with parameter values before running it. However, you're doing this in one shot anyway, so why not the better syntax of:

oc new-app -f <template-path>/rhpam-70-authoring.yaml -p PARAMETER=value -p PARAMETER=value